trlx/trainer/accelerate_base_trainer.py

```diff
@@ -308,6 +308,7 @@ def save_pretrained(self, directory: Optional[str] = None, **kwargs):
     def save(self, directory: Optional[str] = None, **kwargs):
         """Creates a checkpoint of the optimizer, scheduler and model"""
         self.accelerator.save_state(directory or self.config.train.checkpoint_dir, **kwargs)
+        self.accelerator.unwrap_model(self.model).config.save_pretrained(directory or self.config.train.checkpoint_dir)
 
     def load(self, directory: Optional[str] = None, **kwargs):
```

Comment:
should we update load function as well?

Reply:
I don't think so - line 311 only adds a config.json to the save directory - everything else comes from save_state